spaces/S000041/properties/P000061.md

```diff
@@ -0,0 +1,16 @@
+---
+space: S000041
+property: P000061
+value: false
+---
+
+Let $U = \bigl( \left[ 0, 1 \right] \cap \mathbb Q \bigr) \times (0, 1) \cup \bigl\{ \left< 0, 0 \right>, \left< 1, 1 \right> \bigr\}$ be an $F_\sigma$ open set in $X$, therefore a cozero set, since {S41|P13} (see Corollary 1.5.13 in {{zb:0684.54001}}).
```

Comment:
```suggestion
The set $U = \bigl( \left[ 0, 1 \right] \cap \mathbb Q \bigr) \times (0, 1) \cup \bigl\{ \left< 0, 0 \right>, \left< 1, 1 \right> \bigr\}$ is an $F_\sigma$ open set in $X$, therefore a cozero set, since {S41|P13} (see Corollary 1.5.13 in {{zb:0684.54001}}).
```